src/StorageSync/StorageSync/CloudEndpoint/InvokeChangeDetectionCommand.cs

```diff
@@ -275,17 +308,16 @@ public override void ExecuteCmdlet()
                     storageSyncServiceName = StorageSyncServiceName;
                 }
 
-                var triggerChangeDetectionParameters = default(TriggerChangeDetectionParameters);
+                var triggerChangeDetectionParameters = new TriggerChangeDetectionParameters();
```

Comment:
Do we have to pass in an empty object like this, or can we simply pass null in the full share case?

Reply:
Either is fine, but passing in null would require a change to the SDK since we don't allow 'parameters' to be null there